opta/commands/destroy.py

```diff
@@ -66,7 +66,7 @@ def destroy(config: str, env: Optional[str], auto_approve: bool) -> None:
         tf_flags.append("-auto-approve")
 
     gen_all(layer)
-    Terraform.init("-reconfigure")
+    Terraform.init(False, "-reconfigure")
```

Comment:
@juandiegopalomino 
I have added the default value here. Let me know if you want to ignore the error instead.